firefox_ui_harness/decorators.py

```diff
@@ -13,3 +14,11 @@ def wrapper(self, *args, **kwargs):
             raise SkipTest("Skipping due to e10s")
         return target(self, *args, **kwargs)
     return wrapper
+
+
+def skip_under_xvfb(target):
+    def wrapper(self, *args, **kwargs):
+        if os.environ.get('MOZ_XVFB'):
+            raise SkipTest("Skipping due to travis CI")
```

Comment:
I wouldn't necessarily add the relation to travis here. Just say it is because of xvfb.